pkg/identityserver/gateway_registry.go

```diff
@@ -434,6 +508,18 @@ func (is *IdentityServer) purgeGateway(ctx context.Context, ids *ttnpb.GatewayId
 	return ttnpb.Empty, nil
 }
 
+func isClaimAuthenticationCodeValid(authCode ttnpb.GatewayClaimAuthenticationCode) error {
```

Comment:
Bit strange that something called `isXXXValid` returns an error instead of a bool. Better to rename to `validateXXX`